README.md

```diff
@@ -450,6 +409,17 @@ class SomeComponent extends React.Component {
 }
 ```
 
+## DevTools
+
+`mobx-react@6` and higher are no longer compatible with the mobx-react-devtools. 
+That is, the MobX react devtools will no longer show render timings or dependency trees of the component. 
+The reason is that the standard React devtools are no also capable of highlighting re-rendering components.
```

Comment:
`devtools are no also capable` - what did you mean here? Looks like a typo.

Reply:
@macbern Use the suggestions next time please
```suggestion
The reason is that the standard React devtools are also capable of highlighting re-rendering components.
```